pramen-py/src/pramen_py/utils/file_system.py

```diff
@@ -21,9 +21,15 @@
 
 import attrs
 
+from pyhocon import ConfigFactory  # type: ignore
 from pyspark.sql import SparkSession
 from typing_extensions import Protocol
 
+from pramen_py.models import InfoDateSettings, MetastoreTable
+
+
+HACON_METASTORE_TABLES_VARIABLE = "pramen.metastore.tables"
```

Comment:
Should this be:
```suggestion
HOCON_METASTORE_TABLES_VARIABLE = "pramen.metastore.tables"
```
?